src/style/components/search-input.less

```diff
@@ -17,31 +17,31 @@
  *
  */
 
-.user-input {
-  position: relative;
-  display: block;
-  border: 1px solid var(--gray-40);
-  background: var(--white);
-  border-radius: 12px;
-  color: var(--main-color);
-
-  &.modal-style {
-    min-height: 48px;
-    margin: 0 -8px;
-    background: var(--app-bg-secondary);
-    color: var(--background);
-
-    .search-outer {
-      padding: 12px 8px;
-
-      .search-icon {
-        width: 28px;
-        margin-right: 6px;
-        text-align: center;
-      }
-    }
-  }
-}
+// .user-input {
```

Comment:
Is it really not needed anymore? I don't think I understand why

Reply:
well I can not find any element with class name of user-input in the entire project...